src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -351,4 +353,14 @@ private function include(string $name, array $context = []): string
 
         return trim(ob_get_clean());
     }
+
+    /**
+     * Allow overriding the default non-debug template.
+     *
+     * @param string $template relative path to the template file to render
```

Comment:
I think an absolute path could work too, isn't it?

Reply:
No an absolute path cannot work because the template is passed to the include function which appends it to a hard coded ` __DIR__.'/../Resources/'`

```php
private function include(string $name, array $context = []): string
    {
        extract($context, EXTR_SKIP);
        ob_start();
        include __DIR__.'/../Resources/'.$name;

        return trim(ob_get_clean());
    }
```

Reply:
I think we should allow it somehow, otherwise it's hard to intuit and prone to error.

Reply:
Cool I'll see what I can do. I was going for the light touch approach before...